docs/rules/always-return.md

```diff
@@ -31,3 +39,51 @@ myPromise.then((b) => {
   }
 })
 ```
+
+#### Options
+
+##### `ignoreLastCallback`
+
+You can pass an `{ ignoreLastCallback: true }` as an option to this rule to the
+last `then()` callback in a promise chain does not warn if it does not have a
+`return`. Default is `false`.
```

Comment:
I think it would be a good idea to default this to `true` in the next major release

Reply:
I also prefer that option to be `true` by default.
But I think the champion's judgment is necessary.

Reply:
Yeah that's why we should do it in a major release, not now